las/client.py

```diff
@@ -718,6 +735,23 @@ def execute_transition(self, transition_id: str) -> Dict:
         endpoint = f'/transitions/{transition_id}/executions'
         return self._make_request(requests.post, endpoint, body={})
 
+    def delete_transition(self, transition_id: str) -> Dict:
+        """Delete the transition with the provided transition_id, calls the DELETE /transitions/{transitionId} endpoint.
```

Comment:
Should we add a comment: "Will fail if transition is in use by one or more workflows"